test/utils/config_test.py

```diff
@@ -0,0 +1,8 @@
+from otri.utils import config as cfg
+import unittest
+
+class ConfigTest(unittest.TestCase):
+
+    def test_get_key_no_exception(self):
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/gh/OTRI-Unipd/OTRI/pullRequest?prid=5763045)